### PR TITLE
rename Cartesi DApp

### PR DESCRIPTION
**Pull request by ZzzzHui**

### CODE REVIEW

**Comment by ZzzzHui** [comment]
Seems the commit message is too long

**Comment by guidanoli** [comment]
Maybe we can use `app` instead of `application` for variable names.

**Comment by guidanoli** [comment]
This will need a rebase.

**Review by guidanoli — APPROVED** [review]

File: onchain/rollups/.changeset/sweet-pianos-exist.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/rollups": major
+---
+
+Replaced `DApp` with `Application` and removed `Cartesi` from contract names.
```
Comment (guidanoli) [inline onchain/rollups/.changeset/sweet-pianos-exist.md:5]: Actually, I would list the new names.

The `CartesiDApp` contract was renamed as `Application`.
The `ICartesiDApp` contract was renamed as `IApplication`.
The `CartesiDAppFactory` contract was renamed as `ApplicationFactory`.
The `DAppAddressRelay` contract was renamed as `ApplicationAddressRelay`.
The `encodeDAppAddressRelay` function from the former `DAppAddressRelay` contract was renamed as `encodeApplicationAddressRelay`.
The `OnlyDApp` error from the former `CartesiDApp` contract was renamed as `OnlyApplication`.

File: onchain/rollups/.changeset/sweet-pianos-exist.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+"@cartesi/rollups": major
+---
+
+The CartesiDApp contract was renamed as Application.
+The ICartesiDApp contract was renamed as IApplication.
+The CartesiDAppFactory contract was renamed as ApplicationFactory.
+The DAppAddressRelay contract was renamed as ApplicationAddressRelay.
+The encodeDAppAddressRelay function from the former DAppAddressRelay contract was renamed as encodeApplicationAddressRelay.
+The OnlyDApp error from the former CartesiDApp contract was renamed as OnlyApplication.
```
Comment (guidanoli) [inline onchain/rollups/.changeset/sweet-pianos-exist.md:10]: Can we format these names as code, please?

Comment (ZzzzHui) [reply onchain/rollups/.changeset/sweet-pianos-exist.md:10]: Right. For sure :)

**Review by guidanoli — DISMISSED** [review]
LGTM :rocket:

**Comment by ZzzzHui** [comment]
rebased

**Review by guidanoli — APPROVED** [review]

**Pull request merged**